index.js

```diff
@@ -1,7 +1,7 @@
-const DirectorySwitcherPlugin = require('./config/webpack/plugin/resolve/DirectorySwitcherPlugin');
-const LocalPackageAliasPlugin = require('./config/webpack/plugin/resolve/LocalPackageAliasPlugin');
-const TerraDevSiteEntrypoints = require('./config/webpack/plugin/TerraDevSiteEntrypoints');
-const TerraDevSite = require('./config/webpack/plugin/TerraDevSite');
+const DirectorySwitcherPlugin = require('./lib/webpack/plugin/resolve/DirectorySwitcherPlugin');
+const LocalPackageAliasPlugin = require('./lib/webpack/plugin/resolve/LocalPackageAliasPlugin');
+const TerraDevSiteEntrypoints = require('./lib/webpack/plugin/TerraDevSiteEntrypoints');
```

Comment:
These are now included in the dev site webpack plugin by default. Should we still export them? They _could_ be used stand alone.... i lean toward removing them.

Reply:
I vote remove them and expose them later on if requested or needed. Less (can) = more.